CMakeLists.txt

```diff
@@ -481,6 +481,26 @@ if(BUILD_BINDINGS_JAVA OR BUILD_BINDINGS_PYTHON)
   add_subdirectory(src/bindings)
 endif()
 
+#-----------------------------------------------------------------------------#
+# Package configuration
+#
+# Install config and configversion for CVC4 for find_package(CVC4).
+
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  ${CMAKE_CURRENT_BINARY_DIR}/CVC4ConfigVersion.cmake
+  VERSION ${CVC4_RELEASE_STRING}
+  COMPATIBILITY SameMajorVersion
+)
+
+# Install the config, configversion and custom find modules
+install(FILES
+  ${CMAKE_CURRENT_LIST_DIR}/cmake/CVC4Config.cmake
```

Comment:
Why aren't we using `configure_package_config_file()` to generate the real `CVC4Config.cmake` file from a template as [recommended](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#command:configure_package_config_file)?

Reply:
Changed it to use `configure_package_config_file()`. Tested it with another cmake project and `find_package(CVC4)` works as expected.